resources/deprecated/README.md

```diff
@@ -0,0 +1,13 @@
+---
+description: Some documentation about Angle that is not relevant anymore
```

Comment:
We can put the deprecated section at the end, like after international, so that it's the last thing people see in the docs